migrations/v2.js

```diff
@@ -0,0 +1,531 @@
+    _.set(course, '_globals._accessibility._ariaLabels.skipNavigation', 'Skip navigation');
+    return true;
+  });
+
+  checkContent('core - check config._logging', async (content) => {
+    const isValid = _.has(config, '_logging');
+    if (!isValid) throw new Error('core - _logging not added');
+    return true;
+  });
+
+  checkContent('core - check config._accessibility._isSkipNavigationEnabled', async (content) => {
+    const isValid = _.has(config, '_accessibility._isSkipNavigationEnabled');
+    if (!isValid) throw new Error('core - _accessibility._isSkipNavigationEnabled not added');
+    return true;
+  });
+
+  checkContent('core - check course._globals._accessibility.skipNavigationText', async (content) => {
+    const isValid = _.has(course, '_globals._accessibility.skipNavigationText');
+    if (!isValid) throw new Error('core - _globals._accessibility.skipNavigationText not added');
+    return true;
+  });
+
+  checkContent('core - check course._globals._accessibility._ariaLabels.skipNavigation', async (content) => {
+    const isValid = _.has(course, '_globals._accessibility._ariaLabels.skipNavigation');
+    if (!isValid) throw new Error('core - _globals._accessibility._ariaLabels.skipNavigation not added');
+    return true;
+  });
+
+  testSuccessWhere('defaults course/config', {
+    content: [
+      {
+        _type: 'course',
+        _globals: {
+          _accessibility: {
+            _ariaLabels: {
+              skipNavigation: 'Skip navigation'
+            },
+            skipNavigationText: 'Skip navigation'
+          }
+        }
+      },
+      {
+        _type: 'config',
+        _accessibility: {
+          _isSkipNavigationEnabled: true
+        },
+        _logging: logging
+      }
+    ]
+  });
+
+  testSuccessWhere('empty course/config', {
+    content: [
+      { _type: 'course' },
+      { _type: 'config' }
+    ]
+  });
+
+  testStopWhere('no course/config', {
+    content: [{ _type: 'other' }]
+  });
+});
+
+describe('core - update to v2.0.18', async () => {
+  let course;
+
+  whereContent('core - where course', async (content) => {
+    course = getCourse();
+    return course;
+  });
+
+  mutateContent('core - update _globals._accessibility._accessibilityToggleTextOn', async (content) => {
+    const originalValue = _.get(course, '_globals._accessibility._accessibilityToggleTextOn');
+    _.set(course, '_globals._accessibility.accessibilityToggleTextOn', originalValue || 'Turn accessibility on?');
+    _.unset(course, '_globals._accessibility._accessibilityToggleTextOn');
+    return true;
+  });
+
+  mutateContent('core - update _globals._accessibility._accessibilityToggleTextOff', async (content) => {
+    const originalValue = _.get(course, '_globals._accessibility._accessibilityToggleTextOff');
+    _.set(course, '_globals._accessibility.accessibilityToggleTextOn', originalValue || 'Turn accessibility off?');
```

Comment:
```suggestion
    _.set(course, '_globals._accessibility.accessibilityToggleTextOff', originalValue || 'Turn accessibility off?');
```
As is, it would override the previous mutation, I think.